app-crypt/libmd/Manifest

```diff
@@ -1 +1,5 @@
+AUX libmd-1.0.4-fix-version-script-linker-support-detection.patch 1055 BLAKE2B 65c8b3042459786247fb1d09258a23be5d591cc007638104a78e2f301289207570d4dfde569a5d2449bb510dcb94f6244f20ddc05e74781b56a6c08c5417c651 SHA512 1ce46c634a645523a3d02827b2277e458d74a5436ab9e5b50f8965f4670dcab365659e121acd17675cea405e3b37582c7a0d55bb21243920daec9d02708210b9
```

Comment:
Please use thin manifests for your local repo if you can, as it'll avoid this noise. We don't need the AUX/EBUILD/etc entries in thin manifest repos (anything with git).

Reply:
Didn't even know about Thin Manifests, my bad. Fixed.